tests/fixtures/encoding-comment.php

```diff
@@ -15,4 +15,4 @@
 
 <?php //-*- coding: utf-8 -*-
 // @phpcsWarningOnPreviousLine CodeQuality.EncodingComment
-?>
\ No newline at end of file
+?>
```

Comment:
Do we need the trailing closing `?>` tag? If not, let's remove it. It's unnecessary, and not present in any other file either.

```suggestion
```